src/www/index.html

```diff
@@ -209,7 +230,7 @@
   <div class="container">
     <form class="form-group generator_form" enctype="multipart/form-data" id="form" method="post" name="form" role="form">
       <div class="form-group col-md-offset-4 col-xs-offset-2 col-xs-8 col-md-4" style="margin-bottom: 0;">
-        <label for="name">Email</label> <input autofocus="" class="form-control" id="email" name="email" required="true" type="email"><br>
+        <label for="name"><font color="#FFFFFF">Email</font></label> <input autofocus="" class="form-control" id="email" name="email" required="true" type="email"><br>
```

Comment:
@samyakbvs Please use a class here for color:
eg 
```
.white-color {
 color: #ffffff;
}
```

Now add the class to the label tag `<label for="name" class="white-color">`

Do the same for other colors